scripts/lang/fr/__init__.py

```diff
@@ -442,6 +442,8 @@
     "déverbal de": 'f"Déverbal de {italic(parts[1])}"',
     # {{dénominal de|affection|fr}}
     "dénominal de": 'f"Dénominal de {italic(parts[1])}"',
+    # {{diminutif|fr|m=1}}
+    "diminutif": f'"Diminutif" if [p for p in parts if p in ("m=1", "m=oui")] else "diminutif"',
```

Comment:
```suggestion
    "diminutif": 'f"Diminutif" if [p for p in parts if p in ("m=1", "m=oui")] else "diminutif"',
```

Reply:
OK last edit should be fine, but I did not test it.

Reply:
Ok. Tested. It works.